icu4c/source/i18n/unicode/numberformatter.h

```diff
@@ -2217,7 +2217,10 @@ class U_I18N_API NumberFormatterSettings {
      * @draft ICU 68
      */
     Derived usage(StringPiece usage) &&;
+#endif // U_HIDE_DRAFT_API
 
+#ifndef U_HIDE_DRAFT_API
+#ifndef U_HIDE_INTERNAL_API
     /**
```

Comment:
The methods following this were @internal ICU 69 tech preview. In addition to adding the #ifndef U_HIDE_INTERNAL_API, I split them into a separate ifndef U_HIDE_DRAFT_API block since they will likely end up with different dtaf versions than the regular @draft ICU 68 methods before them.